R_D_HW_3/homework3.py

```diff
@@ -0,0 +1,14 @@
+print('I love Python')
```

Comment:
This code shouldn't be here since it is not related to homework 4

Reply:
I'm sorry, I was in a rush doing homework in a case of electcity situation, my file had another path in configuraion parametrs, but I'm already find how to fix it